app/schema/portal/cdo/manage.js

```diff
@@ -6,7 +6,8 @@ const manageCdosGetschema = Joi.object({
 
 const manageCdosQueryschema = Joi.object({
   sortOrder: Joi.string().valid('ASC', 'DESC'),
-  sortKey: Joi.string().valid('indexNumber', 'cdoExpiry', 'owner', 'policeForce', 'joinedExemptionScheme', 'interimExemptFor')
+  sortKey: Joi.string().valid('indexNumber', 'cdoExpiry', 'owner', 'policeForce', 'joinedExemptionScheme', 'interimExemptFor'),
+  noCache: Joi.boolean().truthy('Y').falsy('Y').default(false)
```

Comment:
You are supplying 'Y' as the param noCache value in the ur, yet you are checking against 'Y' in the schema ??